### DISCUSSION TITLE
Converison Not Working

### DISCUSSION THREAD

**Discussion by BobtheBob123456** [Kas-tle/java2bedrock.sh#126] [Converter Help] [answered]
Hey Man,
It's me again.

So I tried converting one of the packs. The only problem is once I did it, it didn't work. 

There were no textures, the custom mapping was blank, and so was pretty much everything else.

I am not sure whether this is a problem with the pack I was trying to convert (https://www.dropbox.com/s/g5rkfdcmlax9w6a/QualityArmoryV2.0.2.zip?dl=1) or the converter itself, but regardless, can you please get to the bottom of this?

Thanks,

**Comment by Kas-tle** [comment, marked as answer]
There is invalid JSON in your pack. See the conversion log: https://github.com/Kas-tle/java2bedrock.sh/runs/7420032233?check_suite_focus=true#step:6:202

**Reply by BobtheBob123456** [reply]
So I fixed that issue, but when I tried it again, it still didn't work. When I looked the logs, I could tell something was wrong, but I couldn't really tell what.

**Reply by typicalsmc** [reply]
can you try to convert using github actions and give me the link again please?

**Reply by typicalsmc** [reply]
![image](https://user-images.githubusercontent.com/53472591/180017578-c0b94b4f-588e-4ddf-88e3-f387db957289.png)
here this that file fixed, missing ,

**Reply by typicalsmc** [reply]
[QualityArmoryV2.0.2.zip](https://github.com/Kas-tle/java2bedrock.sh/files/9151125/QualityArmoryV2.0.2.zip)

**Reply by typicalsmc** [reply]
https://github.com/Kas-tle/java2bedrock.sh/actions/runs/2705727658
^

**Comment by BobtheBob123456** [comment]
Thank you.
